rrtmgp/kernels-openacc/mo_gas_optics_kernels.F90

```diff
@@ -596,15 +601,23 @@ subroutine compute_Planck_source(                        &
     !
     !$acc parallel loop
     do icol = 1, ncol
-      call interpolate1D(tsfc(icol), temp_ref_min, totplnk_delta, totplnk, planck_function(1:nbnd,1,icol))
+      call interpolate1D(tsfc(icol)      , temp_ref_min, totplnk_delta, totplnk, planck_function(1:nbnd,1,icol))
+      call interpolate1D(tsfc(icol) + dST, temp_ref_min, totplnk_delta, totplnk, planck_function(1:nbnd,2,icol))
     end do
+    !$acc parallel loop collapse(2)
```

Comment:
Would it make sense to fuse this loop with the one below and write sfc_source_Jac(igpt,icol) = pfrac(igpt,sfc_lay,icol) * (planck_function(gpoint_bands(igpt), 2, icol) - planck_function(gpoint_bands(igpt), 1, icol)) 
?

Reply:
Agreed. Done